DependencyInjection/Configuration.php

```diff
@@ -81,6 +79,8 @@ public function getConfigTreeBuilder()
                             ->scalarNode('command')->defaultValue(null)->end()
                             ->scalarNode('connection')->defaultValue(null)->end()
                             ->scalarNode('queue')->defaultValue(null)->end()
+                            ->booleanNode('exclusive_processor_stack')->defaultValue(false)->end()
+                            ->append($this->addProcessorStack())
```

Comment:
Sorry for the delay @metfan & btw, thanks for contributing. :/

It fails because there is a `processor` config key defined line 79. The `processor` is transformed in `processors_stack` and it fails because the `processor` key is required (not sure my explanations are really clear).

@stof I'm not sure this `fixXmlConfig` is a good idea right now. All processors in the stack are created by the stack builder thanks to their class name. The main processor is instantiated by the user.
Having only 1 array containing processors in the stack + the main processor could not be very intuitive.

I really need to take some time to update this stack system according to @adrienbrault's recommendations to be able to have something more consistent and easy to use.

What's your point?